torch/functional.py

```diff
@@ -89,37 +89,77 @@ def lu_unpack(LU_data, LU_pivots, unpack_data=True, unpack_pivots=True):
         unpack_data (bool): flag indicating if the data should be unpacked
         unpack_pivots (bool): flag indicating if the pivots should be unpacked
 
-    Example::
+    Examples::
```

Comment:
Does this render correctly?

Reply:
Yes, I presume so because `Examples::` is used also elsewhere. For instance,
```
$ grep -r 'Examples::' torch | grep 'Examples' | wc
    165     315    7198
```